nuntium/migrations/0065_rename_variables_without_writeit.py

```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+    def forwards(self, orm):
+        for template in orm.ConfirmationTemplate.objects.all():
+            template.content_text = template.content_text.replace(u'{writeit_name}', u'{site_name}')
+            template.content_text = template.content_text.replace(u'{writeit_url}', u'{site_url}')
+            template.content_html = template.content_html.replace(u'{writeit_name}', u'{site_name}')
+            template.content_html = template.content_html.replace(u'{writeit_url}', u'{site_url}')
+            template.subject = template.subject.replace(u'{writeit_name}', u'{site_name}')
+            template.subject = template.subject.replace(u'{writeit_url}', u'{site_url}')
+            template.save()
+
+        for template in orm.NewAnswerNotificationTemplate.objects.all():
+            template.template_text = template.template_text.replace(u'{writeit_name}', u'{site_name}')
+            template.template_text = template.template_text.replace(u'{writeit_url}', u'{site_url}')
+            template.template_html = template.template_html.replace(u'{writeit_name}', u'{site_name}')
+            template.template_html = template.template_html.replace(u'{writeit_url}', u'{site_url}')
+            template.subject_template = template.subject_template.replace(u'{writeit_name}', u'{site_name}')
+            template.subject_template = template.subject_template.replace(u'{writeit_url}', u'{site_url}')
+            template.save()
```

Comment:
If we're completely removing the {writeit_*} substitutions then we'll presumably need to change any  time they appear in anyone's templates.

Personally, I'd probably have gone for introducing new variables, and changing all the defaults & documentation etc to them, kept the old ones working for anything that's currently using them…